v2/fleet/docker-cleanup.service

```diff
@@ -10,11 +10,8 @@ User=core
 Restart=on-failure
 RestartSec=20
 TimeoutStartSec=0
-ExecStart=/usr/bin/bash -c '/usr/bin/wget -O /home/core/docker-cleanup.sh \
-https://raw.githubusercontent.com/behance/mesos-systemd/master/v2/util/docker-cleanup.sh \
-&& /usr/bin/bash /home/core/docker-cleanup.sh'
+ExecStart=/usr/bin/sudo bash /home/`whoami`/mesos-systemd/v2/util/docker-cleanup.sh
```

Comment:
@chr0n1x pointed out that User=core is defined above this line.  Does `$HOME/mesos-systemd/v2/util/docker-cleanup.sh` work?